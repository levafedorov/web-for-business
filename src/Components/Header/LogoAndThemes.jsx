import React, {Component} from 'react';
import ThemeBar from './ThemeBar';
import Positioner from '../Page/Positioner';

export default class LogoAndThemes extends Component{
   
    render(){
        return <section className="logo-and-themes">
                  <Positioner>
                        <div className="logo">YOURCOMPANYNAME</div>
                        <ThemeBar/>
                  </Positioner>
             </section>
    }
}