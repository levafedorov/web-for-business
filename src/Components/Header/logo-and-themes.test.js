import React from 'react';
import {shallow} from 'enzyme';
import LogoAndThemes from './LogoAndThemes.jsx';


test('Should logo render', () => {
    const component = shallow(<LogoAndThemes />);
    expect(component).toMatchSnapshot();
});