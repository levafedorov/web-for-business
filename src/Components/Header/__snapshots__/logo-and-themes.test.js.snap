// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should logo render 1`] = `
<section
  className="logo-and-themes"
>
  <Positioner>
    <div
      className="logo"
    >
      YOURCOMPANYNAME
    </div>
    <ThemeBar />
  </Positioner>
</section>
`;
